theme-release-100rc1.rst

```diff
@@ -0,0 +1,115 @@
+`Sphinx`_ 4, and fixes several issues stemming from `docutils`_ 0.17.
+
+.. _sphinx_rtd_theme: https://github.com/readthedocs/sphinx_rtd_theme
+.. _released to PyPI: https://pypi.org/project/sphinx-rtd-theme/1.0.0rc1/
+.. _issue-tracker: https://github.com/readthedocs/sphinx_rtd_theme/issues
+
+.. _Sphinx: https://pypi.org/project/Sphinx/
+.. _docutils: https://pypi.org/project/docutils/
+
+Highlights
+----------
+
+Here are some of the most significant changes included in release 1.0.0:
+
+Support dropped for old dependencies
+    Support is removed for Sphinx versions 1.6 or older, and for Python versions
+    3.0 to 3.3. Theme developers no longer test using these versions and
+    compatibility is not guaranteed for these versions.
+
+HTML4 support will be dropped soon
+    Support for the Sphinx HTML4 writer will be deprecated in release 2.0.
+
+Installation from source won't be supported soon
+    Tentatively scheduled for release 3.0, installation from source will no
+    longer be a supported installation method. Currently, users are installing
+    directly from our GitHub repository, which complicates development of static
+    assets.
+
+Support for new dependencies
+    The theme now supports Sphinx 4 and docutils 0.17, both the latest in each
+    respective release series.
+
+New accessibility features
+    Also added were a number of accessibility updates, including better keyboard
+    navigation support and more descriptive navigation for screen readers.
+
+Added 4 new translations 
+    The theme is close to being almost fully translated in 14 different
+    languages. This will provide project maintainers with localized navigation
+    elements and accessibility content.
+
+`The theme roadmap <roadmap_>`_ has more information on upcoming releases, and backwards
+incompatible changes for each. A full list of changes included in this release
+is available in `the theme changelog <changelog_>`_.
+
+.. _roadmap: https://sphinx-rtd-theme.readthedocs.io/en/latest/development.html#roadmap
+.. _changelog: https://sphinx-rtd-theme.readthedocs.io/en/latest/changelog.html
+
+Testing
+-------
+
+The 1.0.0rc1 release is a pre-release package, meaning it will not install by
+default as the most recent release. Project maintainers will need to specify the
+release version explicitly to install the release candidate.
+
+In your package's ``extra_requirements``, or a separate ``requirements.txt``
+file, you can pin this dependency with:
+
+.. code::
+
+    sphinx_rtd_theme==1.0.0rc1
+
+If you notice any new, unwanted behavior, feel free to open an issue
+`on our issue tracker <issue-tracker_>`_.
+
+Upcoming changes
+----------------
+
+We have several upcoming releases planned, covered in more depth in
+`the theme roadmap <roadmap_>`_.
+
+Of note, our 2.0 release will be dropping support for a number of old
+dependencies, including Sphinx versions < 3.0, Sphinx HTML4 writer support,
+and Internet Explorer 11.
```

Comment:
This is also mentioned in "Highlights" as well. Now that I read this section, we could probably remove it from "Highlights" to avoid duplicated content?

Reply:
I tried reducing, but the only duplicated content is note of the HTML4 removal. This paragraph is pretty succinct already